script.js

```diff
@@ -1,4 +1,141 @@
+//Game: Beat That!
+//2 players, 2 dices
+//2 game modes: Each player rolls once and player picks the order of the numbers rolled to get the biggest value
+//Player with higher combined number wins
+//Calculate number of wins
+
+//2 players
+var player1 = "Player 1";
+var player2 = "Player 2";
+var player1DiceRoll = [];
+var player2DiceRoll = [];
+var player1Number;
+var player2Number;
+var player1WinCount = 0;
+var player2WinCount = 0;
+var player1LoseCount = 0;
+var player2LoseCount = 0;
+var drawCount = 0;
+
+//2 game modes, start with first player
+var modeRollDice = "modeRollDice";
+var modeChooseDiceOrder = "modeChooseDiceOrder";
+var currentPlayer = "Player 1";
+
+//Game starts with rolling of dice
+var gameMode = modeRollDice;
+
+var rollSingleDice = function () {
+  return Math.floor(Math.random() * 6) + 1;
+};
+
+var rollDiceAndAssign = function () {
+  var getDiceNumber = [rollSingleDice(), rollSingleDice()];
+
+  if (currentPlayer === "Player 1") {
```

Comment:
ahh yes! A `===` strict equality operator!

It actually is a more recommended way of doing as opposed to a `==` equality operator.

the only difference being the `===` checks the data type as well as the value. as opposed to a `==`.
i.e.
```
1=="1" //true
1==="1" //false
```